pom.xml

```diff
@@ -688,6 +688,7 @@
             <effort>Max</effort>
             <failOnError>true</failOnError>
             <includeTests>true</includeTests>
+            <maxHeap>1024</maxHeap>
```

Comment:
Does this help solve a build problem, or were you just trying different things and left this in?